src/components/commonElement.tsx

```diff
@@ -64,6 +64,18 @@ export const EmptyButton = styled.button`
   }
 `;
 
+export const CrossButton = styled(EmptyButton)`
```

Comment:
what's the criteria whether a component goes in the `commonElement` or in the `components` folder?

I think putting it here make it less discoverable and reusable. My suggestion would be to have all components in the `components` folder

Reply:
commonElements are actually the list of styled-components not normal components
Will do it in a new PR if needed
This would lead to lot of changes